css/styles.css

```diff
@@ -0,0 +1,99 @@
+body {
+  background-color: #ede8a7;
+}
+
+h1 {
+  font-size: 50px;
+  line-height: 0px;
+  font-family: 'Bahiana', cursive;
+}
+
+h2 {
+  font-size: 15px;
+  color: #6b3328;
+  text-transform: uppercase;
+  font-family: 'Roboto 700', sans-serif;
+  letter-spacing: 5px;
+}
+
+header {
+  width: 100%;
+  padding-left: 50px;
+  padding-top: 50px;
+
+}
+
+p, ul {
+  font-size: 20px;
+  line-height: 30px;
+  font-family: 'Roboto', sans-serif;
+}
+
+
+.ListLink {
+  font-size: 20px;
+  line-height: 30px;
+  font-family: 'Roboto', sans-serif;
+  color: orange;
+  text-decoration: none;
+} 
+
+
+a {
+  font-size: 30px;
+  font-family: 'Bahiana', cursive;
+  color: #b28231;
+}
+
+a:hover {
+  color: #205529;
+}
+
+article {
+  border-left: 5px black solid;
+  width: 50%;
+  padding: 50px;
+  float: left;
+}
+
+aside {
+  width: 30%;
+  padding-left: 50px;
+  padding-top: 50px;
+  float: left;
+}
+
+footer {
+  float: left;
+  padding: 50px;
+}
+
+div#likes {
+  float: left;
+  align-content: center;
+  width: 40%;
+  padding: 50px;
+}
+
+div#dislikes {
+  float: left;
+  align-content: center;
```

Comment:
`align-content` is used with the parent container has a display property of `flex`.